src/__tests__/__utils__/validator.test.js

```diff
@@ -16,10 +16,35 @@ describe("Validator Class", () => {
       password: "Password is required"
     });
   });
+
   it("should validate matching of two values", () => {
     expect(Validator.isMatch("password", "1234", "1234ad")).toEqual({
       message: "Your passwords  do not match!"
     });
     expect(Validator.isMatch("password", "1234", "1234")).toEqual({});
   });
+
+  it("should validate new article", () => {
```

Comment:
@YvesIraguha, I think the team's conventions were to use `test()` function instead of `it()` any other ones. Should you please replace them?